.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Purpose
+_Describe the problem or feature in addition to a link to the issues._
+
+## Approach
+_How does this change address the problem?_
+
+## Testing
+_How did you test your changes?_
+
+#### Open Questions and Pre-Merge TODOs
+- [ ] Use github checklists. When solved, check the box and explain the answer.
+
+## Learning
+_Describe the research stage_
+
+_Links to blog posts, patterns, libraries or addons used to solve this problem_
+
+#### Blog Posts
```

Comment:
Should we make this heading more general, or add similar sections for documentation / stackoverflow?

Reply:
Hmmm, "Blog Posts" is meant to be a subheading for "Learning". I'll merge it in and we can update it if need be.